src/modules/process.c

```
@@ -1363,6 +1363,45 @@ static jsval_t env_to_object(ant_t *js, jsval_t *args, int nargs) {
+static jsval_t env_toString(ant_t *js, jsval_t *args, int nargs) {  
+  size_t buf_cap = 4096;
+  char *buf = malloc(buf_cap);
+  size_t pos = 0;
+  
+  for (char **env = environ; *env != NULL; env++) {
+    char *entry = *env;
+    size_t entry_len = strlen(entry);
+    
+    if (pos + entry_len + 2 >= buf_cap) {
+      buf_cap = buf_cap * 2 + entry_len;
+      buf = realloc(buf, buf_cap);
```

Cppcheck (2.19.0)

[error] Common realloc mistake

(memleakOnRealloc)